### PR TITLE
Add support for binder

### PR DESCRIPTION
**Pull request by zuphilip**
The link in the badge will work as soon as this PR is accepted and be part of your master branch. For now this can tested on this link: https://mybinder.org/v2/gh/zuphilip/MOSM_talk/binder?filepath=00reproducible_manuscripts.Rmd&urlpath=rstudio

### CODE REVIEW

File: install.R [hunk 1]
```diff
@@ -0,0 +1,7 @@
+install.packages("tidyverse")
+install.packages("effsize")
+install.packages("prmisc")
+install.packages("binb")
```
Comment (einGlasRotwein) [inline install.R:4]: This line can be deleted, right? Because `prmisc` is installed from GitHub later and this line will fail anyways?

**Comment by einGlasRotwein** [comment]
Thanks a lot for the PR! Great that you solved the issue. I only have one question about the installation file. I guess `install.packages("prmisc")` can be deleted because it doesn't do anything anyways?

File: install.R (hunk 1 above)
Comment (zuphilip) [reply install.R:4]: Ah you mean to delete the line
```diff
-install.packages("prmisc")
```
Yes, sounds reasonable!

**Comment by zuphilip** [comment]
Currently, you have the following line https://github.com/einGlasRotwein/MOSM_talk/blob/518013fbb8c88dd3953b45654fe2526d6276385f/00reproducible_manuscripts.Rmd#L12

and for that you need the package to install first. If you do not need the library at all then you should delete this line first in your RMarkdown file and can then adiust the Dockerfile.

**Pull request merged**